CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Ctrl-C no longer kills background jobs for which job control is disabled, matching POSIX semantics (#6828).
 - Improve Gradle completion
 - Fixed `pushd`'s behavior with respect to the directory stack when given an invalid argument
+- A new environment variable, `$fish_force_vi_cursor`, has been added. This can be set to force `fish_vi_cursor` to change the cursor shape in vi mode.
```

Comment:
Maybe drop `environment`, this variable doesn't need to be exported in most scenarios. Also the motivation could be extended by adding something like "without regard to your terminal" at the end.

Reply:
Done.